branch update

sambacha opened the pull request:
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

vpicone committed:
chore: trigger failure

vpicone committed:
ci: update commitlint workflow

vpicone committed:
docs: update commitlint docs

vpicone committed:
Merge pull request #1032 from vpicone/add-commitlint

hi: add commitlint enforcement to Husky

vpicone committed:
chore: adjust release-it config

vpicone committed:
chore: add commit message for ci

vpicone committed:
chore: release 1.26.18

vpicone committed:
ci: add release workflow

vpicone committed:
Update manual.yml

vpicone committed:
Update manual.yml

vpicone committed:
Update package.json

vpicone committed:
ci: restrict to release managers

vpicone committed:
ci: fix manual workflow

vpicone committed:
docs: update release instructions

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/freight-trust/gatsby-theme-carbon-base?pullRequest=8) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/freight-trust/gatsby-theme-carbon-base?pullRequest=8) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/freight-trust/gatsby-theme-carbon-base?pullRequest=8) it.</sub>

Commit:
chore(merge): merge from master

jnm2377 committed:
feat: update switcher content (#1023)

* feat: update switcher content

* fix: consolidate divider margin styles

* fix: design link

* fix: community order and ibm.com label

* fix: add back iX

* fix: ibm.com link

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

Commit:
Merge branch 'main' of github.com:carbon-design-system/gatsby-theme-carbon into main

luizamendes committed:
feat(Aside): add option to hide hanging rule from Aside component (#1014)

* Adding option to hide hanging rule from Aside component

* Update packages/gatsby-theme-carbon/src/components/Aside/Aside.module.scss

* Adding example of new prop hideRule

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vp@vincepic.one>

sstrubberg committed:
feat(Theming): Adds Dark, White, and g10 themes to GTC (#979)

* themeing works, now i just need design direction

* merge from master

* playing with colors

* theme'in

* custom header, custom homepage, custom innerpages

* rough draft for docs

* g10 interior theme

* white mode components looking sharp

* Update configuration.mdx

Removed all caps in Favicon content (IMPORTANT to Important)

* knocking out more components with color tokens

* cleaned up leftnav, stripped header theme'ing

* clean up zombies

* removed old sidenav styles and modifiers

* page header border

* updated the api per vince suggestion

* default themeing

* fix: edit link

* fix: swap resource loading order

* fix: header link color

* fixed up code snippet

* page-table and code blocks

* fixed resourcecard tokens and code blocks

* returned to default themeing

* removed a consolelog

* default theme

* Update packages/example/src/pages/guides/configuration.mdx

Co-authored-by: Josefina Mancilla <32556167+jnm2377@users.noreply.github.com>

* Update packages/example/src/pages/guides/configuration.mdx

Co-authored-by: Josefina Mancilla <32556167+jnm2377@users.noreply.github.com>

* removed zombie code from anchorlinks

* fixed the font color for the homepage header

* fixed all the code prism styles

* back to g10 interior

* chore: ran yarn

* style: fixed colors on PageHeader and PageTabs

* style: changed back to default theme

* style: removed custom PageHeader style

* style: switching to dark mode for reviews

* style: icon color for resource cards in dark mode

* style: switched back to dark mode

* style: replaced icons to prevent themeing conflict

* style: notification fixes for dark mode

* docs: new images that are themeing agnostic

* fix: vince feedbacks

* style: knocking out leftnav importants

* style: added comments for overriding styles, replaced images on demo page

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>
Co-authored-by: Scott Strubberg <sstrubberg@pm.me>
Co-authored-by: Mike Abbink <32881239+mjabbink@users.noreply.github.com>
Co-authored-by: Vince Picone <vp@vincepic.one>
Co-authored-by: Josefina Mancilla <32556167+jnm2377@users.noreply.github.com>

vpicone committed:
Update manual.yml

vpicone committed:
ci: update workflow npm auth

vpicone committed:
chore: permit node 14

vpicone committed:
chore: update lock file

vpicone committed:
chore: update release-it and release-it plugins

vpicone committed:
Update manual.yml

vpicone committed:
ci: update manual flow

vpicone committed:
ci: update manual flow

vpicone committed:
ci: build command

vpicone committed:
Update manual.yml

Commit:
Merge branch 'main' of github.com:carbon-design-system/gatsby-theme-carbon into main

vpicone committed:
chore: ensure tags are pulled down

vpicone committed:
ci: update manual flow

vpicone committed:
chore: update git hook

vpicone committed:
Merge branch 'main' of https://github.com/carbon-design-system/gatsby-theme-carbon into main

vpicone committed:
chore: update git hook

Commit:
Merge branch 'main' of github.com:carbon-design-system/gatsby-theme-carbon into main

Commit:
chore: release 1.27.0

vpicone committed:
chore: use npm instead of yarn

sstrubberg committed:
style(switcher): fixed tokens for dark theme (#1036)

* style(switcher): fixed tokens for dark theme

* chore: default back to dark/g10

* style(footer): backgroud color

Co-authored-by: Scott Strubberg <sstrubberg@pm.me>

vpicone committed:
Update and rename manual.yml to manual-release.yml

vpicone committed:
Update package.json

vpicone committed:
ci: don't pull upstream in ci

vpicone committed:
ci: don't check npm w/ release-it

vpicone committed:
ci: move skip config to yarn workspace plugin

vpicone committed:
chore: ensure we're passing through className props to allow for easier style overrides (#1029)

* feat: add classnames functionality to MDX components and cleanup classname library usaage

chore: add more pass throughs

* refactor: update normalize classname package usage

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

jnm2377 committed:
fix: ordered list styles in firefox  (#1047)

* chore: update carbon dep

* feat: update ordered list styles

sstrubberg committed:
style(search): color tokens for themeing (#1043)

* style(search): color tokens for themeing

* chore: removed dark mode

Co-authored-by: Scott Strubberg <sstrubberg@pm.me>
Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

alisonjoseph committed:
fix(footer): width of rules on mobile (#1051)

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

alisonjoseph committed:
fix(search): update search styles (#1050)

* fix(search): update search styles

* fix: search alignment

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

vpicone committed:
Delete .kodiak.toml

kai687 committed:
docs: fix typo in npm-scripts.mdx (#1046)

* docs: fix typo in npm-scripts.mdx

The word _Sites_ at the beginning should be plural, and not _Site's_.

* fix(footer): width of rules on mobile (#1051)

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>
Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vpicone@gmail.com>

alisonjoseph committed:
fix: feedback and back to top button mobile spacing (#1052)

* fix: feedback and back to top button mobile spacing

* Update packages/gatsby-theme-carbon/src/components/FeedbackDialog/LaunchButton.module.scss

Co-authored-by: Vince Picone <vp@vincepic.one>

Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vp@vincepic.one>

vpicone committed:
fix: default exports

andreancardona committed:
fix(page-tabs): nav aria unique labels (#1063)

* fix(page-tabs): nav aria unique labels

* fix(global search): 1989-aat-avt1-gs-about-carbon-aria-expanded - updated

Co-authored-by: Vince Picone <vpicone@gmail.com>

vpicone committed:
chore: update deps (#1067)

* chore: update deps

* fix: favicon

vpicone committed:
chore: release 1.27.1

andreancardona committed:
fix(global search): 1989-aat-avt1-gs-about-carbon-aria-expanded (#1064)

* fix(global search): 1989-aat-avt1-gs-about-carbon-aria-expanded

* fix(global search): 1989-aat-avt1-gs-about-carbon-aria-expanded - move aria expanded to button

* fix(global search): move aria expanded to button

vpicone committed:
fix: un-minify homepage image to remove jank (#1071)

vpicone committed:
Update getTheme.js

sstrubberg committed:
Update Header.module.scss (#1076)

sstrubberg committed:
style(headernav): active state (#1044)

* style(headernav): active state

* style: removed blue line for sub-items

* style: better descriptors

* style(header): making things more like ibm.com/design

* style(header): making things more like ibm.com/design

* Update packages/gatsby-theme-carbon/src/components/Homepage/homepage.scss

Co-authored-by: Vince Picone <vpicone@gmail.com>

* style(header): active states

* style(header): border on sub-menu items

* style(headernav): border problems

* fix border in dropdown header

* style: removed zombies and wrote better commentary

* style: removed last border in a menu

* style: seperated styles into different modules

* style: removed zombies

* style: organized styles for readability

* style: extra commentary

Co-authored-by: Scott Strubberg <sstrubberg@pm.me>
Co-authored-by: kodiakhq[bot] <49736102+kodiakhq[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vpicone@gmail.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

sstrubberg committed:
Update gatsby-config.js

Commit:
chore: release 1.27.2

jnm2377 committed:
fix: 404 link color and config branch as main (#1085)

* fix: change config to main branch

* fix: change 404 link color token

barlock committed:
fix: fail gracefully without page titles (#1065)

* fix: fail gracefully without page titles

* fix: add support for custom title back

* feat: automatically generate page titles from file path

* chore: revert null check in nextprevious

Co-authored-by: Josefina Mancilla <32556167+jnm2377@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump ini from 1.3.5 to 1.3.7 (#1075)

Bumps [ini](https://github.com/isaacs/ini) from 1.3.5 to 1.3.7.
- [Release notes](https://github.com/isaacs/ini/releases)
- [Commits](https://github.com/isaacs/ini/compare/v1.3.5...v1.3.7)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Josefina Mancilla <32556167+jnm2377@users.noreply.github.com>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

sstrubberg committed:
style: fixes z-index when search is open (#1090)

sstrubberg committed:
fix: added Racial Equity to switcher -closes #1057 (#1088)

Co-authored-by: Scott Strubberg <sstrubberg@pm.me>

sstrubberg committed:
style: removed the padding -closes #1074 (#1077)

* style: removed the padding -closes #1074

* style: padding loss only effects mobile screens

* style: removed extra style, moved copy out of lists

* style: removed extra style

Co-authored-by: Scott Strubberg <sstrubberg@pm.me>

sstrubberg committed:
chore: release 1.27.3

Commit:
Update ResourceCard.mdx (#1097)

Add `none` as a valid option for actionIcon attribute.

sstrubberg committed:
fix: zindex for search bar on headernav style (#1096)

* fix: zindex for search bar on headernav style

* fix: bettering naming

* fix: added some color for maintainers

sstrubberg committed:
Update gatsby-config.js

sstrubberg committed:
feat: date in frontmatter resolves last modifed date on page (#1094)

* feat: date in frontmatter resolves last modifed date on page

* fix: removed zombie

* fix: updated the example

* fix: added toLocalDateString logic

* fix: updated language in the docs

* fix: new component name

vpicone committed:
fix: adjust search container width to align with page contents (#1105)

* fix: adjust search container width to align with page contents closes #1101

* Update GlobalSearch.module.scss

vpicone committed:
Update vercel.json

jnm2377 committed:
fix: update latest carbon and add style override for sidenav overlay bug (#1106)

* chore: update latest carbon components/react

* fix: add sidenav overlay style override

* chore: update latest carbon

vpicone committed:
chore: update deps (#1112)

Commit:
Fix: update package.json scripts for npm users (#1103)

* Fix: update package.json scripts for npm users

The script for `dev:clean` is currently `yarn build && yarn clean` or something like this. 

If a user is utilizing npm as a package manager instead of yarn, or don't have yarn installed on their machine, they will continue to see error when running `npm run dev:clean`.

* fix(package.json): update script to support npm

vpicone committed:
chore: add universal design link closes #1100 (#1113)

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

alisonjoseph committed:
Add sketch kit download (#1114)

vpicone committed:
chore: update release docs

vpicone committed:
chore: release 1.28.0

vpicone committed:
Update commitlint.yml

Palke committed:
feat: adding workplace design into switcher (#1117)

vpicone committed:
chore: update carbon, remove switcher tooltip (#1119)

* chore: update carbon, remove switcher tooltip

* chore: add tooltip back

vpicone committed:
chore: release 1.29.0

vpicone committed:
chore: switch switcher tooltip text

vpicone committed:
chore: pass through switcher tooltip prop

vpicone committed:
chore: release 1.29.1

vpicone committed:
fix: remove light prop (#1126)

* chore: upgrade deps

* fix: remove light prop

vpicone committed:
chore: release 1.29.2

dependabot[bot] committed:
build(deps): bump hosted-git-info from 2.8.8 to 2.8.9 (#1124)

Bumps [hosted-git-info](https://github.com/npm/hosted-git-info) from 2.8.8 to 2.8.9.
- [Release notes](https://github.com/npm/hosted-git-info/releases)
- [Changelog](https://github.com/npm/hosted-git-info/blob/v2.8.9/CHANGELOG.md)
- [Commits](https://github.com/npm/hosted-git-info/compare/v2.8.8...v2.8.9)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump ssri from 6.0.1 to 6.0.2 (#1122)

Bumps [ssri](https://github.com/npm/ssri) from 6.0.1 to 6.0.2.
- [Release notes](https://github.com/npm/ssri/releases)
- [Changelog](https://github.com/npm/ssri/blob/v6.0.2/CHANGELOG.md)
- [Commits](https://github.com/npm/ssri/compare/v6.0.1...v6.0.2)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vpicone@gmail.com>

dependabot[bot] committed:
build(deps): bump lodash from 4.17.20 to 4.17.21 (#1123)

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.20 to 4.17.21.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.20...4.17.21)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Vince Picone <vpicone@gmail.com>

vpicone committed:
feat: add option to remove site switcher from header (#1132)

* feat: add option to disable switcher

* docs: add documentation for switcher disabling

vpicone committed:
chore: release 1.30.0

vpicone committed:
fix: cover gap between page tabs and shell

vpicone committed:
chore: release 1.30.1

aledavila committed:
chore(404): makes links visible in dark container (#1139)

* chore(404): makes links visible in dark container

* chore(404): remove important add globals

* chore(404): remove styling from g10

jeffchew committed:
chore(switcher): changing to new url for Carbon for IBM.com (#1142)

vpicone committed:
chore: release 1.30.2

vpicone committed:
Next (#1138)

* chore: update gatsby and react

* fix: sassOptions

* chore: update gatsby plugins and remove emotion

* fix: sass module import

* chore: remove emotion

* chore: move homepage to its own component

* chore: add pre-release script

* chore: remove console logs

* breaking: migrate to gatsby v3 (#1118)

* chore: update gatsby and react

* fix: sassOptions

* chore: update gatsby plugins and remove emotion

* fix: sass module import

* chore: remove emotion

* chore: move homepage to its own component

* chore: add pre-release script

* chore: remove console logs

* chore: use major pre-release

* chore: npm tag

* chore: release 2.0.0-next.0

* chore: update deps

* chore: don't force major script

* chore: release 2.0.0-next.1

* chore: release 2.0.0-next.2

* chore: more dep updates

* chore: release 2.0.0-next.3

* fix: useLayoutEffect issues

* fix: useLayoutEffect warnings (#1136)

* chore: release 2.0.0-next.4

* docs: add v2 migration guide

* docs: update migration guide

vpicone committed:
chore: update deps (#1144)

vpicone committed:
chore: release 2.0.0

Commit:
feat(shadowing): update guidelines for shadowing header and footer (#1148)

Commit:
docs(migration): update migration docs to include shadowing (#1150)

* docs(migration): update migration docs to include shadowing

* docs(migration): mispelling

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

folkforms committed:
chore: Fix path for shadowing example (#1157)

grahamharper committed:
fix: changes theme_color to gray 100 (#1164)

grahamharper committed:
Update npm-scripts docs to match actual scripts (#1163)

emyarod committed:
feat(minicard): support transferring props to Gatsby Links and anchors (#1160)

* feat(minicard): support transferring props to Gatsby Links and anchors

* chore: format

* docs(minicard): fix typo

Co-authored-by: Vince Picone <vpicone@gmail.com>

Commit:
feat(square-card): add new square card component (#1151)

* feat(square-card): create square card

* chore(square-card): update card

* feat(square-card): update card styling and add example page

* chore(square-card): update page info

* chore(package): add pictograms package

* feat(square-card): fix mobile styling and add dark and pictogram props

* chore(square-card): add pictogram example

* chore(square-card): changed title sizes for large screens

* chore(square-card): css changes

* feat(square-card): changed styles to module styles

vpicone committed:
chore: release 2.1.0

alisonjoseph committed:
fix: update grid padding at large breakpoint (#1167)

alisonjoseph committed:
fix: switcher button focus state (#1170)

alisonjoseph committed:
fix: line length for lists and headings (#1172)

Co-authored-by: Vince Picone <vpicone@gmail.com>

alisonjoseph committed:
fix: add back active switcher styles (#1175)

* fix: add back active switcher styles

* fix: add hover styles back

* fix: active state

vpicone committed:
chore: update dependencies (#1174)

* chore: update dependencies

* fix: remove non-existant classname

As of https://github.com/carbon-design-system/gatsby-theme-carbon/pull/1170, we're no longer creating a switcherButtonOpen class

* fix: revert 'fix'

vpicone committed:
chore: release 2.1.1

vpicone committed:
chore: check release docs

vpicone committed:
chore: update theme meta colors

vpicone committed:
chore: release 2.1.2

vpicone committed:
fix: resource card border closes #1171 (#1177)

vpicone committed:
chore: release 2.1.3

donovanglover committed:
feat: add node 16 support (#1178)

vpicone committed:
fix: switcher repairs (#1179)

vpicone committed:
chore: release 2.1.4-next.0

vpicone committed:
chore: release 2.1.4

vpicone committed:
fix: medium post image aspect ratio closes #1176 (#1185)

* fix: medium post image aspect ratio closes #1176

* fix: only add background to the bottom half of article cards

alisonjoseph committed:
fix: grid alignment at max breakpoint (#1181)

* fix: grid alignment at max breakpoint

* fix: update to 424px because math

* fix: grid at large xlg breakpoint

vpicone committed:
chore: release 2.1.5

alisonjoseph committed:
Update CODEOWNERS (#1190)

alisonjoseph committed:
Update v2-migration.mdx (#1193)

* Update v2-migration.mdx

* update docs

alisonjoseph committed:
Update Switcher.js (#1192)

Commit:
chore: release 2.1.6

dependabot[bot] committed:
build(deps): bump semver-regex from 3.1.2 to 3.1.3 (#1196)

Bumps [semver-regex](https://github.com/sindresorhus/semver-regex) from 3.1.2 to 3.1.3.
- [Release notes](https://github.com/sindresorhus/semver-regex/releases)
- [Commits](https://github.com/sindresorhus/semver-regex/commits)

---
updated-dependencies:
- dependency-name: semver-regex
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

alisonjoseph committed:
fix: aside docs (#1202)

alisonjoseph committed:
feat(switcher): add carbon for ibm products remove ix (#1205)

Commit:
chore: release 2.2.0

dependabot[bot] committed:
build(deps): bump postcss from 7.0.35 to 7.0.36 (#1206)

Bumps [postcss](https://github.com/postcss/postcss) from 7.0.35 to 7.0.36.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/7.0.35...7.0.36)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump engine.io from 4.1.1 to 4.1.2 (#1207)

Bumps [engine.io](https://github.com/socketio/engine.io) from 4.1.1 to 4.1.2.
- [Release notes](https://github.com/socketio/engine.io/releases)
- [Changelog](https://github.com/socketio/engine.io/blob/4.1.2/CHANGELOG.md)
- [Commits](https://github.com/socketio/engine.io/compare/4.1.1...4.1.2)

---
updated-dependencies:
- dependency-name: engine.io
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

rodet committed:
docs(page-description): update PageDescription docs (#1195)

I've seen people trying to put too much into the description. I think it is a sane guidance to limit the content to one or two short sentences.

Co-authored-by: Alessandra Davila <aledavila@ibm.com>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

dependabot[bot] committed:
build(deps): bump follow-redirects from 1.14.4 to 1.14.7 (#1208)

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.4 to 1.14.7.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.4...v1.14.7)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

dependabot[bot] committed:
build(deps): bump nanoid from 3.1.29 to 3.2.0 (#1210)

Bumps [nanoid](https://github.com/ai/nanoid) from 3.1.29 to 3.2.0.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.1.29...3.2.0)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump trim-off-newlines from 1.0.1 to 1.0.3 (#1213)

Bumps [trim-off-newlines](https://github.com/stevemao/trim-off-newlines) from 1.0.1 to 1.0.3.
- [Release notes](https://github.com/stevemao/trim-off-newlines/releases)
- [Commits](https://github.com/stevemao/trim-off-newlines/compare/v1.0.1...v1.0.3)

---
updated-dependencies:
- dependency-name: trim-off-newlines
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

rodet committed:
fix: update the references to the new homepage (#1212)

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

aledavila committed:
feat(expressive-list): creates the expressive content only list (#1197)

* feat(expressive-list): creates the expressive content only list

* feat(expressive-list): add new props

* feat(expressive-list): add pictogram variant

* chore(expressive-list): clean up

* chore(expressive-list): fix expressive list styles

* docs(expressive-list): fix expressive list docs

* docs(expressive-list): update expressive list docs

* chore(expressive-list): add title type examples

* docs(expressive-list): add title examples to pictogram

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

alisonjoseph committed:
Update CONTRIBUTING.md (#1215)

alisonjoseph committed:
feat: add additional aspect ratios for do dont (#1216)

* feat: add additional aspect ratios for do dont

* chore: add examples for aspect ratios

dependabot[bot] committed:
build(deps): bump follow-redirects from 1.14.7 to 1.14.8 (#1222)

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.7 to 1.14.8.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.7...v1.14.8)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

alisonjoseph committed:
fix: do dont component updates (#1220)

* fix: do dont component updates

* fix: spelling

* fix: add color token

xlegs committed:
feat: new config option to add default mdx layouts (#1214)

* feat: new config option to add default mdx layouts

* feat: add documentation

Co-authored-by: Stefan Zecevic <szecevic@ibm.com>
Co-authored-by: Alessandra Davila <adavila91@gmail.com>

Commit:
feat(expressive-list-container): adds expressive list container component (#1198)

* feat(expressive-list-container): adds expressive list container component

* chore(expressive-list-container): change children prop type

* docs(expressive-list-container): add disclaimer

* Update packages/example/src/pages/components/ExpressiveListContainer.mdx

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

dependabot[bot] committed:
build(deps): bump simple-get from 3.1.0 to 3.1.1 (#1223)

Bumps [simple-get](https://github.com/feross/simple-get) from 3.1.0 to 3.1.1.
- [Release notes](https://github.com/feross/simple-get/releases)
- [Commits](https://github.com/feross/simple-get/compare/v3.1.0...v3.1.1)

---
updated-dependencies:
- dependency-name: simple-get
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Alison Joseph <alison.joseph@us.ibm.com>

Commit:
chore: release 2.3.0

dependabot[bot] committed:
build(deps): bump minimist from 1.2.5 to 1.2.6 (#1235)

Bumps [minimist](https://github.com/substack/minimist) from 1.2.5 to 1.2.6.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.5...1.2.6)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
fix(link): correct link to Cirrus getting started (#1237)

janhassel committed:
fix(expressive-list-container): remove margin-top from first content element (#1240)

tay1orjones committed:
chore: release 2.3.1

tay1orjones committed:
chore(project): replace node-sass with sass (#1257)

tay1orjones committed:
chore: release 2.3.2

krook committed:
Fixed a typo in "extremly" (#1249)

I didn't update the date though, as it was a minor change and didn't affect the content. Please let me know if I should do that.

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

tay1orjones committed:
chore(project): add issue triage automation (#1260)

tay1orjones committed:
chore(release): merge `next` into `main` (#1266)

* breaking: upgrade gatsby dependencies to v4

* chore: release 3.0.0-next.0

* chore(theme): update to 4.4 (#1203)

* fix: medium post image aspect ratio closes #1176 (#1185)

* fix: medium post image aspect ratio closes #1176

* fix: only add background to the bottom half of article cards

* fix: grid alignment at max breakpoint (#1181)

* fix: grid alignment at max breakpoint

* fix: update to 424px because math

* fix: grid at large xlg breakpoint

* chore: release 2.1.5

* Update CODEOWNERS (#1190)

* Update v2-migration.mdx (#1193)

* Update v2-migration.mdx

* update docs

* Update Switcher.js (#1192)

* chore: release 2.1.6

* build(deps): bump semver-regex from 3.1.2 to 3.1.3 (#1196)

Bumps [semver-regex](https://github.com/sindresorhus/semver-regex) from 3.1.2 to 3.1.3.
- [Release notes](https://github.com/sindresorhus/semver-regex/releases)
- [Commits](https://github.com/sindresorhus/semver-regex/commits)

---
updated-dependencies:
- dependency-name: semver-regex
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* fix: aside docs (#1202)

* chore(theme): update to 4.4

Co-authored-by: Vince Picone <vpicone@gmail.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>
Co-authored-by: Vince Picone <vp@vincepic.one>
Co-authored-by: Ale Davila <aledavila@ibm.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* chore: release 3.0.0-next.1

* chore(next): update with latest from main (#1226)

* feat: V11 migration for gatsby-theme (#1209)

* fix: medium post image aspect ratio closes #1176 (#1185)

* fix: medium post image aspect ratio closes #1176

* fix: only add background to the bottom half of article cards

* fix: grid alignment at max breakpoint (#1181)

* fix: grid alignment at max breakpoint

* fix: update to 424px because math

* fix: grid at large xlg breakpoint

* chore: release 2.1.5

* Update CODEOWNERS (#1190)

* Update v2-migration.mdx (#1193)

* Update v2-migration.mdx

* update docs

* Update Switcher.js (#1192)

* chore: release 2.1.6

* build(deps): bump semver-regex from 3.1.2 to 3.1.3 (#1196)

Bumps [semver-regex](https://github.com/sindresorhus/semver-regex) from 3.1.2 to 3.1.3.
- [Release notes](https://github.com/sindresorhus/semver-regex/releases)
- [Commits](https://github.com/sindresorhus/semver-regex/commits)

---
updated-dependencies:
- dependency-name: semver-regex
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* fix: aside docs (#1202)

* feat(switcher): add carbon for ibm products remove ix (#1205)

* chore: release 2.2.0

* build(deps): bump postcss from 7.0.35 to 7.0.36 (#1206)

Bumps [postcss](https://github.com/postcss/postcss) from 7.0.35 to 7.0.36.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/7.0.35...7.0.36)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps): bump engine.io from 4.1.1 to 4.1.2 (#1207)

Bumps [engine.io](https://github.com/socketio/engine.io) from 4.1.1 to 4.1.2.
- [Release notes](https://github.com/socketio/engine.io/releases)
- [Changelog](https://github.com/socketio/engine.io/blob/4.1.2/CHANGELOG.md)
- [Commits](https://github.com/socketio/engine.io/compare/4.1.1...4.1.2)

---
updated-dependencies:
- dependency-name: engine.io
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* docs(page-description): update PageDescription docs (#1195)

I've seen people trying to put too much into the description. I think it is a sane guidance to limit the content to one or two short sentences.

Co-authored-by: Alessandra Davila <aledavila@ibm.com>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

* build(deps): bump follow-redirects from 1.14.4 to 1.14.7 (#1208)

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.4 to 1.14.7.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.4...v1.14.7)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

* feat: migrate to v11

* feat: migrate to v11 yarn lock

* feat: update component prefix

* feat: update imports

* feat: icon size prop/imports

* fix: try to add styles

* build(deps): bump nanoid from 3.1.29 to 3.2.0 (#1210)

Bumps [nanoid](https://github.com/ai/nanoid) from 3.1.29 to 3.2.0.
- [Release notes](https://github.com/ai/nanoid/releases)
- [Changelog](https://github.com/ai/nanoid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ai/nanoid/compare/3.1.29...3.2.0)

---
updated-dependencies:
- dependency-name: nanoid
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* build(deps): bump trim-off-newlines from 1.0.1 to 1.0.3 (#1213)

Bumps [trim-off-newlines](https://github.com/stevemao/trim-off-newlines) from 1.0.1 to 1.0.3.
- [Release notes](https://github.com/stevemao/trim-off-newlines/releases)
- [Commits](https://github.com/stevemao/trim-off-newlines/compare/v1.0.1...v1.0.3)

---
updated-dependencies:
- dependency-name: trim-off-newlines
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* fix: update the references to the new homepage (#1212)

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

* feat: migrate styles to v11

* feat(expressive-list): creates the expressive content only list (#1197)

* feat(expressive-list): creates the expressive content only list

* feat(expressive-list): add new props

* feat(expressive-list): add pictogram variant

* chore(expressive-list): clean up

* chore(expressive-list): fix expressive list styles

* docs(expressive-list): fix expressive list docs

* docs(expressive-list): update expressive list docs

* chore(expressive-list): add title type examples

* docs(expressive-list): add title examples to pictogram

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

* Update CONTRIBUTING.md (#1215)

* feat: import all styles from react

* feat: add additional aspect ratios for do dont (#1216)

* feat: add additional aspect ratios for do dont

* chore: add examples for aspect ratios

* feat(grid): update grid in theme

* build(deps): bump follow-redirects from 1.14.7 to 1.14.8 (#1222)

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.7 to 1.14.8.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.7...v1.14.8)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

* fix: do dont component updates (#1220)

* fix: do dont component updates

* fix: spelling

* fix: add color token

* feat: new config option to add default mdx layouts (#1214)

* feat: new config option to add default mdx layouts

* feat: add documentation

Co-authored-by: Stefan Zecevic <szecevic@ibm.com>
Co-authored-by: Alessandra Davila <adavila91@gmail.com>

* feat(expressive-list-container): adds expressive list container component (#1198)

* feat(expressive-list-container): adds expressive list container component

* chore(expressive-list-container): change children prop type

* docs(expressive-list-container): add disclaimer

* Update packages/example/src/pages/components/ExpressiveListContainer.mdx

Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

* build(deps): bump simple-get from 3.1.0 to 3.1.1 (#1223)

Bumps [simple-get](https://github.com/feross/simple-get) from 3.1.0 to 3.1.1.
- [Release notes](https://github.com/feross/simple-get/releases)
- [Commits](https://github.com/feross/simple-get/compare/v3.1.0...v3.1.1)

---
updated-dependencies:
- dependency-name: simple-get
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Alison Joseph <alison.joseph@us.ibm.com>

* chore: release 2.3.0

* feat(grid): add 12 grid

* fix: fleex grid 12 col

* feat: migrate notification and remove comments

* fix: tile components and aspect ratio

* fix: temp comment out abbr export

* fix(header): update header svg colors

* chore: update package

* chore: update yarn lock

Co-authored-by: Vince Picone <vpicone@gmail.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>
Co-authored-by: Vince Picone <vp@vincepic.one>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Stephane Rodet <s@rodet.org>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>
Co-authored-by: Josefina Mancilla <josefinanoemi4@gmail.com>
Co-authored-by: Alison Joseph <alison.joseph@us.ibm.com>
Co-authored-by: Stefan Zecevic <stefan@zecevic.xyz>
Co-authored-by: Stefan Zecevic <szecevic@ibm.com>

* chore: release 3.0.0-next.2

* chore: release 3.0.0-next.2

* chore(project): update carbon deps (#1228)

* chore: release 3.0.0-next.3

* Remove all usages of carbon components on `next` (#1229)

* fix(v11): remove all usages of carbon-components

* chore(gatsby): update to v4.9.2

* fix(inlinenotification): re-enable inline links (#1231)

* chore: release 3.0.0-next.4

* chore(project): update to carbon 10.56.0, correct style import order (#1234)

* chore(project): update to carbon 10.55.2, correct style import order

* fix(feedbackdialog): correct tokens to fix style issues

* fix(feedbackdialog): style fixes for v11

* chore(project): update carbon deps to 10.56.0

* fix(components): use individual imports instead of central entrypoint

* chore: release 3.0.0-next.5

* fix(overlays): correct z-index for backtotop and launch buttons

* chore: release 3.0.0-next.6

* fix(codeblock): use correct font-family

* chore: release 3.0.0-next.7

* chore(project): update carbon dependencies, related fixes

* chore: release 3.0.0-next.8

* chore(project): update carbon to v11, related import/usage fixes

* chore: release 3.0.0-next.9

* feat(type): update carbon deps, ensure type-classes are emitted with proper prefix

* chore: release 3.0.0-next.10

* fix(grid): add _grid.scss and move over flex grid for precedence (#1242)

* chore: release 3.0.0-next.11

* fix(markdown): ensure links within lists do not have padding (#1246)

* fix(markdown): ensure links within lists do not have uneccessary padding and render inline

* fix(list): update prefix

* chore: release 3.0.0-next.12

* chore(project): update changelog

* feat(gtc): upgrade to v4 Gatsby, v11 Carbon, and associated docs (#1261)

* feat(gtc): upgrade to v4 Gatsby, v11 Carbon, and associated docs

* Update packages/example/src/pages/guides/v3-migration.mdx

Co-authored-by: Taylor Jones <tay1orjones@users.noreply.github.com>

* fix(icon): dropped the file in the right folder

* docs(migration): add icons migration info

* feat(gtc): split docs into two sets of migrations, updated deps round 1

* feat(gtc): updated the rest of the gatsby deps

* chore(gtc): updated carbon deps

* fix(util): removed layout

Co-authored-by: Taylor Jones <tay1orjones@users.noreply.github.com>
Co-authored-by: Taylor Jones <taylor.jones826@gmail.com>

* chore(deps): update carbon deps

* chore: release 3.0.0-next.13

Co-authored-by: Vince Picone <vp@vincepic.one>
Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>
Co-authored-by: Vince Picone <vpicone@gmail.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>
Co-authored-by: Ale Davila <aledavila@ibm.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Scott Strubberg <sstrubberg@us.ibm.com>
Co-authored-by: Stephane Rodet <s@rodet.org>
Co-authored-by: Josefina Mancilla <josefinanoemi4@gmail.com>
Co-authored-by: Alison Joseph <alison.joseph@us.ibm.com>
Co-authored-by: Stefan Zecevic <stefan@zecevic.xyz>
Co-authored-by: Stefan Zecevic <szecevic@ibm.com>
Co-authored-by: DAK <40970507+dakahn@users.noreply.github.com>

tay1orjones committed:
chore: release 3.0.0

tay1orjones committed:
fix(minicard): export from root index.js (#1272)

tay1orjones committed:
Update CODEOWNERS

dependabot[bot] committed:
build(deps): bump semver-regex from 3.1.3 to 3.1.4 (#1245)

dependabot[bot] committed:
build(deps): bump moment from 2.29.1 to 2.29.4 (#1251)

eng618 committed:
feat(yarn): upgrade to v2 (#1276)

* feat(yarn): upgrade to v2

- revert node version change
- update yarn.lock
- add node version manager
- feat: add upgrade-interactive plugin
- chore: remove yarn as a dep
- docs: updates

closes #1275

eng618 committed:
fix(font): use gatsby for static file hosting (#1281)

* fix(font): use gatsby for static file hosting

Set up static file hosting natively through Gatsby rather than deploy
platform.

* chore: remove vercel files

alisonjoseph committed:
Revert "fix(font): use gatsby for static file hosting (#1281)" (#1283)

This reverts commit d012c62592e31b92a7f30dbf65eb95647cee7ac4.

eng618 committed:
build(deps): update node to 18 (#1278)

* build(deps): update node to 18

Also update all other dependencies

- update all minor dependencies

https://github.com/carbon-design-system/gatsby-theme-carbon/issues/1277

* chore: bump everything to node 18

* chore: update gatsby-plugin-sass

* chore: update readme with status badge

* docs: typo correction

* docs: update structure

* ci: update .github/workflows/manual-release.yml

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

* ci: update commitlint version and docs

Co-authored-by: Scott Strubberg <sstrubberg@protonmail.com>

eng618 committed:
Revert "build(deps): update node to 18 (#1278)" (#1284)

This reverts commit 1188ca291010ede2007f642286b52e2cb8e64322.

eng618 committed:
chore: release 3.1.0

dependabot[bot] committed:
build(deps): bump decode-uri-component from 0.2.0 to 0.2.2 (#1285)

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump json5 from 1.0.1 to 1.0.2 (#1287)

Bumps [json5](https://github.com/json5/json5) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v1.0.1...v1.0.2)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Eric Garcia <eng618@garciaericn.com>

eng618 committed:
build(deps): bump all minor and below versions (#1292)

* build(deps): bump all minor and below versions

* build: bump node version

* build: bump a couple more

eng618 committed:
build(deps): bump all minor and below (#1299)

eng618 committed:
chore: release 3.1.1

jeffchew committed:
chore(banner): add banner message for community contributions (#1307)

* chore(banner): add banner message for community contributions

* chore(banner): update to messaging

---------

Co-authored-by: jeffchew <jeff.chew@ibm.com>

eng618 committed:
build(deps): bump all minor and below (#1311)

eng618 committed:
chore: release 3.1.2

alisonjoseph committed:
chore: update Carbon deps (#1315)

alisonjoseph committed:
chore: release 3.1.3

alisonjoseph committed:
chore: release 3.1.3

alisonjoseph committed:
chore: update all Carbon dependencies (#1318)

alisonjoseph committed:
chore: release 3.1.4

dependabot[bot] committed:
build(deps): bump semver from 5.7.1 to 5.7.2 (#1314)

Bumps [semver](https://github.com/npm/node-semver) from 5.7.1 to 5.7.2.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/v5.7.2/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v5.7.1...v5.7.2)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump ua-parser-js from 0.7.32 to 0.7.35 (#1312)

Bumps [ua-parser-js](https://github.com/faisalman/ua-parser-js) from 0.7.32 to 0.7.35.
- [Release notes](https://github.com/faisalman/ua-parser-js/releases)
- [Changelog](https://github.com/faisalman/ua-parser-js/blob/master/changelog.md)
- [Commits](https://github.com/faisalman/ua-parser-js/compare/0.7.32...0.7.35)

---
updated-dependencies:
- dependency-name: ua-parser-js
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump socket.io-parser from 4.2.2 to 4.2.4 (#1308)

Bumps [socket.io-parser](https://github.com/socketio/socket.io-parser) from 4.2.2 to 4.2.4.
- [Release notes](https://github.com/socketio/socket.io-parser/releases)
- [Changelog](https://github.com/socketio/socket.io-parser/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io-parser/compare/4.2.2...4.2.4)

---
updated-dependencies:
- dependency-name: socket.io-parser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Alison Joseph <alisonejoseph@Gmail.com>

carbon-automation[bot] committed:
chore(release): update carbon deps (#1323)

Co-authored-by: alisonjoseph <alisonjoseph@users.noreply.github.com>

eng618 committed:
build: update deps (#1322)

tay1orjones committed:
Delete issue-triage.yml (#1286)

eng618 committed:
chore: release 3.1.5

dependabot[bot] committed:
build(deps): bump @sideway/formula from 3.0.0 to 3.0.1 (#1290)

Bumps [@sideway/formula](https://github.com/sideway/formula) from 3.0.0 to 3.0.1.
- [Release notes](https://github.com/sideway/formula/releases)
- [Commits](https://github.com/sideway/formula/compare/v3.0.0...v3.0.1)

---
updated-dependencies:
- dependency-name: "@sideway/formula"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: update dependabot configuration

dependabot[bot] committed:
build(deps): bump actions/checkout from 2 to 3 (#1327)

Bumps [actions/checkout](https://github.com/actions/checkout) from 2 to 3.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v2...v3)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump wagoid/commitlint-github-action from 2.1.0 to 5.4.3 (#1325)

Bumps [wagoid/commitlint-github-action](https://github.com/wagoid/commitlint-github-action) from 2.1.0 to 5.4.3.
- [Changelog](https://github.com/wagoid/commitlint-github-action/blob/master/CHANGELOG.md)
- [Commits](https://github.com/wagoid/commitlint-github-action/compare/v2.1.0...v5.4.3)

---
updated-dependencies:
- dependency-name: wagoid/commitlint-github-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump actions/setup-node from 2.pre.beta to 3.7.0 (#1326)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 2.pre.beta to 3.7.0.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2-beta...v3.7.0)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump mkdirp from 1.0.4 to 3.0.1 (#1330)

dependabot[bot] committed:
build(deps): bump word-wrap from 1.2.3 to 1.2.5 (#1319)

dependabot[bot] committed:
build(deps): bump webpack from 5.75.0 to 5.76.1 (#1293)

carbon-automation[bot] committed:
chore(release): update carbon deps (#1333)

Co-authored-by: guidari <guidari@users.noreply.github.com>

eng618 committed:
chore: release 3.1.6

eng618 committed:
ci: add upto release workflow (#1335)

* ci: add upto release workflow

* build: update lock

eng618 committed:
ci: quick update on git config

eng618 committed:
ci: git config user hardcoded

eng618 committed:
ci: update release workflow

eng618 committed:
ci: update release workflow

eng618 committed:
chore: let get derive push url

eng618 committed:
ci: configure ssh

eng618 committed:
chore: release 3.1.7

eng618 committed:
build(deps): update minor and below (#1336)

* build(deps): bump all minor and below

* chore: update engines

* build(deps): update minor and below

* build: rebuild lock file

eng618 committed:
chore: release 3.1.8 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1337)

Co-authored-by: tay1orjones <tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 3.1.9 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1338)

Co-authored-by: guidari <guidari@users.noreply.github.com>

eng618 committed:
chore: release 3.1.10 [skip ci],

dependabot[bot] committed:
build(deps): bump stylelint and stylelint-scss (#1334)

Bumps [stylelint](https://github.com/stylelint/stylelint) and [stylelint-scss](https://github.com/stylelint-scss/stylelint-scss). These dependencies needed to be updated together.

Updates `stylelint` from 13.13.1 to 15.10.3
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/13.13.1...15.10.3)

Updates `stylelint-scss` from 3.21.0 to 5.1.0
- [Release notes](https://github.com/stylelint-scss/stylelint-scss/releases)
- [Changelog](https://github.com/stylelint-scss/stylelint-scss/blob/master/CHANGELOG.md)
- [Commits](https://github.com/stylelint-scss/stylelint-scss/compare/3.21.0...v5.1.0)

---
updated-dependencies:
- dependency-name: stylelint
  dependency-type: direct:production
  update-type: version-update:semver-major
- dependency-name: stylelint-scss
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 3.1.11 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1339)

Co-authored-by: alisonjoseph <alisonjoseph@users.noreply.github.com>

eng618 committed:
chore: release 3.1.12 [skip ci],

eng618 committed:
chore: bump deps and format code

eng618 committed:
chore: release 3.1.13 [skip ci],

dependabot[bot] committed:
build(deps): bump eslint-plugin-react-hooks from 2.5.1 to 4.6.0 (#1341)

Bumps [eslint-plugin-react-hooks](https://github.com/facebook/react/tree/HEAD/packages/eslint-plugin-react-hooks) from 2.5.1 to 4.6.0.
- [Release notes](https://github.com/facebook/react/releases)
- [Changelog](https://github.com/facebook/react/blob/main/packages/eslint-plugin-react-hooks/CHANGELOG.md)
- [Commits](https://github.com/facebook/react/commits/HEAD/packages/eslint-plugin-react-hooks)

---
updated-dependencies:
- dependency-name: eslint-plugin-react-hooks
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

eng618 committed:
chore: release 3.1.14 [skip ci],

eng618 committed:
ci: add workflow to validate lint and format (#1345)

* ci: add workflow to validate lint and format

* style: apply formatting

* chore: cleanup lint errors and warnings

* Revert "build(deps): bump stylelint and stylelint-scss (#1334)"

This reverts commit 071efb601804e82a91faed59da58fe971c5c201f.

* fix: correct formmat change

* ci: require build-and-test before release

* ci: rename for clarity

* ci: rename for clarity

* chore: test => validate

eng618 committed:
ci: update release workflow

eng618 committed:
chore: release 3.1.15 [skip ci],

eng618 committed:
ci: update release workflow

eng618 committed:
build: update yarn.lock

eng618 committed:
build(deps): bump scss related

eng618 committed:
ci: update release workflow

eng618 committed:
chore: release 3.1.16 [skip ci],

eng618 committed:
docs: add badge to README

eng618 committed:
build: update yarn.lock

eng618 committed:
chore: release 3.1.17 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1346)

Co-authored-by: tw15egan <tw15egan@users.noreply.github.com>

eng618 committed:
chore: release 3.1.18 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1347)

Co-authored-by: tay1orjones <tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 3.1.19 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1348)

Co-authored-by: tay1orjones <tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 3.1.20 [skip ci],

eng618 committed:
build(deps): bump all minor and below (#1355)

* build: rebuild lock file

* build(deps): bump all minor and below

eng618 committed:
chore: release 3.1.21 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1356)

Co-authored-by: andreancardona <andreancardona@users.noreply.github.com>

eng618 committed:
chore: release 3.1.22 [skip ci],

carbon-automation[bot] committed:
chore(release): update carbon deps (#1360)

Co-authored-by: tay1orjones <tay1orjones@users.noreply.github.com>

eng618 committed:
chore: release 3.1.23 [skip ci],
